src/style.css

```diff
@@ -1 +1,37 @@
 /* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(images/star.svg);
+
+  display: flex;
+}
+
+.stars__star + .stars__star {
+  margin-left: 4px;
+}
```

Comment:
Consider adding a fallback font-family for better cross-browser compatibility. Even though this block does not set the `font-family` directly, it's a good practice to ensure that all text-containing elements have a fallback font specified.